README.md

```diff
@@ -34,6 +34,8 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+The project requires a number of secret keys in the .env file. Therse are API_KEY, API_TOKEN and TRELLO_BOARD_ID.
```

Comment:
Could you add placeholders (not real values!) to `.env.template` too?

It also wouldn't hurt to have a little more detail here on what API_KEY/API_TOKEN are, and that you've assumed the board has "To Do", "Doing" and "Done" columns.

Reply:
Placeholders added to .env.template.

More detail added to the README file.